test/unit/pipeline/FetchUnitTest.cc

```diff
@@ -6,6 +6,7 @@
 #include "gtest/gtest.h"
 #include "simeng/Instruction.hh"
 #include "simeng/arch/Architecture.hh"
+#include "simeng/kernel/Linux.hh"
```

Comment:
Given that `#include "simeng/arch/Architecture.hh"` already includes `#include "simeng/kernel/Linux.hh"`, this include should be redundant.

Reply:
Removed